.github/workflows/autobuild.yml

```diff
@@ -216,29 +204,23 @@ jobs:
             /opt/android/android-ndk
           key:                      ${{ matrix.config.target_os }}-${{ hashFiles('.github/workflows/autobuild.yml', 'autobuild/android/autobuild_apk_1_prepare.sh', 'autobuild/android/install-qt.sh') }}-${{ matrix.config.cmd1_prebuild }}
 
-      # Prepare (install QT & dependencies)
       - name:                       "Prepare for ${{ matrix.config.config_name }}"
```

Comment:
What about this:

```suggestion
      - name:                       "Install build dependencies for ${{ matrix.config.config_name }}"
```

Reply:
Yes. I think that's better (which would also imply a change of all the script file names...)